packages/remix-dev/compiler/utils/postcss.ts

```diff
@@ -119,12 +120,17 @@ async function loadTailwindPlugin(
 
   try {
     // First ensure they have a Tailwind config
-    require.resolve("./tailwind.config", { paths: [rootDirectory] });
+    // tailwind doesn't support esm config files yet
+    let tailwindConfigExtensions = [".js", ".cjs"];
+    let tailwindConfig = findConfig(
+      rootDirectory,
+      "tailwind.config",
+      tailwindConfigExtensions
+    );
+    if (!tailwindConfig) throw new Error("No Tailwind config found");
 
     // Load Tailwind from the project directory
-    tailwindPath = require.resolve("tailwindcss", {
-      paths: [rootDirectory],
-    });
+    tailwindPath = require.resolve("tailwindcss", { paths: [rootDirectory] });
   } catch (err) {
```

Comment:
we're not doing anything with the caught error.

```suggestion
  } catch {
```